exercises/dnd-character/metadata.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Randomly generate Dungeons & Dragons characters"
+source: "Simon Shine, Erik Schierboom"
```

Comment:
While I appreciate the reference, this is really all your work, so feel free to remove my name.

Reply:
Oh, but I wouldn't have suggested this as an exercise at all; it was just an example.

And your current feedback has more than warranted it. I'd like to think of it as a joint effort. :-)